aspnetcore/signalr/version-differences.md

```diff
@@ -24,12 +34,12 @@ ASP.NET Core SignalR supports JSON, as well as a new binary protocol based on [M
 
 ## Differences on the server
 
-The SignalR server-side libraries are included in the `Microsoft.AspNetCore.App` package that is part of the **ASP.NET Core Web Application** template for both Razor and MVC projects.
+The ASP.NET Core SignalR server-side libraries are included in the `Microsoft.AspNetCore.App` package that is part of the **ASP.NET Core Web Application** template for both Razor and MVC projects.
 
-SignalR is an ASP.NET Core middleware, so it must be configured by calling `AddSignalR` in `Startup.ConfigureServices`.
+ASP.NET Core SignalR is an ASP.NET Core middleware, so it must be configured by calling `AddSignalR` in `Startup.ConfigureServices`.
```

Comment:
If we have an API broswer link for "AddSignalR," I recommend linking it.

Reply:
Yep. We didn't at the time this was original authored but I'll add it now.